src/graph/graph-ops.ts

```diff
@@ -45,13 +48,13 @@ export const highlightNodesBatch = (ids: string[]) => (graph: Graph) => ({
     if (!ids.includes(node.id)) {
       return {
         ...node,
-        data: { ...node.data, highlight: false },
+        data: { ...node.data, style: undefined },
       }
     }
 
     return {
       ...node,
-      data: { ...node.data, highlight: !node.data.highlight },
+      data: { ...node.data, style: node.style?.color ? undefined : { color: HIGHLIGHT_COLOR } },
```

Comment:
This is actually nice to have style instead of boolean `highlight` and `color` separately. I would have event put `node.style` as a required property in `DataPayload`